xml/SectorCategory.xml

```diff
@@ -435,7 +435,7 @@
             <code>910</code>
             <name>
                 <narrative>ADMINISTRATIVE COSTS OF DONORS</narrative>
-                <narrative xml:lang="fr">Frais administratifs (non alloués par secteur)</narrative>
+                <narrative xml:lang="fr">ADMINISTRATIVE COSTS OF DONORS</narrative>
```

Comment:
language change (this looks like a mistake at the DAC CRS end)

Reply:
Thanks, it seems to me they have aligned the category name in English and have kept the sub-category in English and French with the correct translation.
![image](https://user-images.githubusercontent.com/24607542/30171490-11388222-93ea-11e7-90e5-af9ec4cea3d9.png)

![image](https://user-images.githubusercontent.com/24607542/30171471-fea58ab0-93e9-11e7-9a23-8974917448da.png)